subsys/bluetooth/shell/audio.c

```diff
@@ -1409,6 +1409,38 @@ static int cmd_term_broadcast_sink(const struct shell *sh, size_t argc,
 }
 #endif /* CONFIG_BT_AUDIO_BROADCAST_SINK */
 
+static int cmd_context(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err = 0;
+	enum bt_audio_dir dir;
+	enum bt_audio_context ctxt;
+
+	ctx_shell = sh;
```

Comment:
You don't need to do it here, the `ctx_shell` is initialized in `bt init`

Reply:
Done